packaging_automation/tests/test_package_cloud_statistics_collector.py

```diff
@@ -68,7 +72,7 @@ def test_fetch_and_save_package_cloud_stats():
 
     records = session.query(PackageCloudDownloadStats).all()
 
-    assert len(records) == filtered_package_count * PACKAGE_SAVED_HISTORIC_RECORD_COUNT
+    assert len(records) > 0
```

Comment:
Is it enough to check for a positive number of records here? If we stop storing new statistics after a while, will this test fail, or will it silently ignore the problem?

Reply:
We are dropping the table before inserting so this check assures record existence